src/extensions/resolveSource.ts

```diff
@@ -16,14 +15,17 @@ export interface RegistryEntry {
  * Gets the source for a given extension name and version from the official extensions registry
  *
  * @param extensionName the name, or name@version of the extension to get the ExtensionSource for
+ * @param errMessage custom error message for when the extensionName is not found
```

Comment:
This is not a common pattern to be following. Is there a particular reason why this is done here?

I would expect the caller of the function to catch and throw a new error if they wanted to change the error message, not pass the error message into the function.

Reply:
I revised, and am now catching the error and throwing new one.